test/fixtures/flag/flag.js

```diff
@@ -0,0 +1,13 @@
+module.exports = () => {
+  return [
+    {
+      title: "dark - mode",
+      enabled: "true",
+      roles: ["members", "no_role", "super_user"],
+      users: {
+        "pallab id": "true",
+        "rohit id": "false",
```

Comment:
```suggestion
        "pcBpdEQ5ydRakHHSpY2p": "true",
        "quEpdE34ydRakJSpY45p": "false",
```

Lets keep it accurate to the real data :)

Reply:
When i put the id then eslint makes it a key instead of string. How do i resolve that?

Reply:
It's a key.

Reply:
then should i make the true/false as boolean as well??

Reply:
Yes.